CHANGELOG.md

```
@@ -3,6 +3,7 @@
 #### 9.4.18 - October 2024
   * Added new `FieldEnum` with all the available field names
   * Code security improvements as per CodeQL recommendations
+  * Added new `MtSequenceInfoEnum` with all the available inner sequences of specific MT schemas
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)